external/storm-jms/src/main/java/org/apache/storm/jms/spout/JmsSpout.java

```diff
@@ -288,19 +393,28 @@ public void fail(Object msgId) {
         }
     }
 
-    public void declareOutputFields(OutputFieldsDeclarer declarer) {
+    /**
+     * {@inheritDoc}
```

Comment:
Nit: I think javadoc does this automagically. I think inheritDoc is in case you want to add some extra javadoc and then inherit the rest